### PR TITLE
LPS-98387 Simplify

### PR DESCRIPTION
**Pull request by achaparro**

### CODE REVIEW

**Comment by achaparro** [comment]
ci:test:sf

**Comment by achaparro** [comment]
ci:test:relevant

**Comment by achaparro** [comment]
ci:test:upgrade

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java [hunk 1]
```diff
@@ -391,12 +391,12 @@ public MBMessage addMessage(
 
 		message.setCategoryId(categoryId);
 		message.setThreadId(thread.getThreadId());
+		message.setTreePath(message.buildTreePath());
```
Comment (nhpatt) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImpl.java]: this has to be called after setParentMessageId

**Pull request closed without merging**